ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -284,6 +286,11 @@ export const WorkspaceCard = fp.flow(withNavigation)(
                     <div style={{ fontSize: 12 }}>
                       Last Changed: {displayDate(workspace.lastModifiedTime)}
                     </div>
+                    {workspaceLibrary && (
```

Comment:
Good to show who created the workspace in featured workspace as there could be multiple owners sharing workspace with same name